idseq_dag/steps/blast_contigs.py

```diff
@@ -99,10 +99,10 @@ def summary(self):
             r["length"] = 1
         r["pident"] = sum(hsp["pident"] * hsp["length"] for hsp in self.optimal_cover) / r["length"]
         r["bitscore"] = sum(hsp["bitscore"] for hsp in self.optimal_cover)
-        r["qstart"] = min(hsp["qstart"] for hsp in self.optimal_cover)
-        r["qend"] = max(hsp["qend"] for hsp in self.optimal_cover)
-        r["sstart"] = min(hsp["sstart"] for hsp in self.optimal_cover)
-        r["send"] = max(hsp["send"] for hsp in self.optimal_cover)
+        r["qstart"] = min(min(hsp["qstart"], hsp["qend"]) for hsp in self.optimal_cover)
+        r["qend"] = max(max(hsp["qstart"], hsp["qend"]) for hsp in self.optimal_cover)
+        r["sstart"] = min(min(hsp["sstart"], hsp["send"]) for hsp in self.optimal_cover)
+        r["send"] = max(max(hsp["sstart"], hsp["send"]) for hsp in self.optimal_cover)
```

Comment:
The above changes are part of another PR that is intended to land before this one.   Both are included here because that would save testing time.